docs/admin/serving/revision-gc.md

```diff
@@ -0,0 +1,96 @@
+# Revision garbage collection
+
+The `config-gc` ConfigMap contains settings that determine in-active revisions are cleaned up. This ConfigMap is located in the `knative-serving` namespace.
+
+
+## Cluster-wide configuration
+
+The following properties allow you to configure revision garbage collection:
+
+-|-
+name|description
+`retain-since-create-time`| Duration since creation before considering a revision for GC or "disabled"
+`retain-since-last-active-time`| Duration since active before considering a revision for GC or "disabled"
+`min-non-active-revisions`| Minimum number of non-active revisions to retain.
+`max-non-active-revisions`| Maximum number of non-active revisions to retain or "disabled" to disable any maximum limit.
```

Comment:
Presumably this should be a table but I think the formatting is wrong since it's not rendering in the preview?
https://deploy-preview-4266--knative.netlify.app/development/admin/serving/revision-gc/#cluster-wide-configuration

Reply:
good catch - let me fix that